contentcuration/contentcuration/frontend/shared/views/channel/mixins.js

```diff
@@ -125,6 +127,7 @@ export const channelExportMixin = {
           channel.aggregators,
           channel.licenses,
           channel.copyright_holders,
+          channel.containsContentFrom,
```

Comment:
Hrm, when I tested this locally, the `contains content from` field was present in the CSV, but empty.

I think `containsContentFrom` may not be the correct key to get this information.

In the Details modal, it references `original_channels` for this information: https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/shared/views/details/Details.vue#L251

But we may want to limit that to just the titles of those channels.

To test this, there should be an 'imported channel' in the default test setup, it should show the 'Published Channel' in this column for its CSV export.

Reply:
I too faced the same issue, that the field was empty. I did try to test it properly but not sure why I couldn't import channels in my locally that were published already. I also tried various ways by creating new published channels, and all that did not work. I will try to fix this by this weekend, haven't been able to give a look at it cos of the final lab-exams and end exams. I will try to fix this by this weekend. Sorry for the delay.

Reply:
Not a problem - we understand you are busy and volunteering your time!

Reply:
Thank you @rtibbles !!